### PR TITLE
Release alec version 3.0.0

### PR DESCRIPTION
**Pull request by BenjaminJ**

### CODE REVIEW

**Review by RangerRick — CHANGES_REQUESTED** [review]
I am honestly not even sure how to "review" a 458-commit merge. I can [look at circleci](https://app.circleci.com/pipelines/github/OpenNMS/alec/1365/workflows/c9740e41-9bc9-4fcb-9d23-46e325cf69f7) and see that the branch you're merging from is green. I can see that the last commit removes `-SNAPSHOT` versioning from things.

Is there something useful I can actually do to understand what's happening here enough to "approve" it rather than just go "uhh I dunno sure"? Or is this just a formality to get around the need to have a review to merge things because github blocks it?

**Comment by BenjaminJ** [comment]
I was following the release.md file, all those commits have already been reviewed, but I still need approval to merge the release into master. I'm asking review to be sure that is the way to go.

**Review by RangerRick — APPROVED** [review]
If it's just for the sake of release, I feel like the solution is to either a) have an admin do the final merge/release, b) make an ALEC team person (presumably you \<g>) the "owner" and promote you to admin for this repo, or c) remove the restrictions on this repo because it's not useful to limit master when there's only a few committers anyway.

This feels like it needs to be a larger conversation about how we handle releases for these projects that are not part of the main horizon/meridian release process.

**Pull request merged**
